### ISSUE TITLE
Chocolatey upgrade fails

### ISSUE THREAD

**Issue by simonsan** [CoatiSoftware/Sourcetrail#1060]
> ERROR: Cannot process argument transformation on parameter 'file'. Cannot convert value to type System.String.
The upgrade of sourcetrail was NOT successful.
> Error while running 'C:\ProgramData\chocolatey\lib\sourcetrail\tools\chocolateyinstall.ps1'.

I would need to deinstall and install the new version. Upgrading is not working.

**Comment by mlangkabel** [comment]
@digitalcoyote: Do you have an idea what could cause this issue?

**Comment by digitalcoyote** [comment]
> @digitalcoyote: Do you have an idea what could cause this issue?

I haven't been able to reproduce it, but I may have some ideas once I get another chance to look at it.

**Comment by mlangkabel** [comment]
I managed to reproduce the issue by following these steps:
* uninstall current version of Sourcetrail from the system
* `choco install sourcetrail --version 2020.1.117`
* `choco upgrade sourcetrail`

**Comment by digitalcoyote** [comment]
Reproduced. I'll see what I can do to fix it.

**Comment by digitalcoyote** [comment]
TL;DR Looks like it's caused by a lack of proper cleanup in the install script... I'll be modifying it and correcting that in a fix Version for Chocolatey. Otherwise it should be corrected with the next Release of SourceTrail

It appears that after unzipping the Release package from Github, it is not cleaned up by the current script. When it circles back around on an upgrade, this leaves 2 `sourcetrail.msi` files as children of the unzip directory. So the script is attempting to pass in the full name of both files instead of the full name of the correct one. The solution will be to properly cleanup after Unzipping. I'll Include a check for now to make sure the folder does not already exist to allow those who already installed via Chocolatey to Upgrade.

**Comment by mlangkabel** [comment]
@digitalcoyote: thanks for looking into it so quickly. So as far as I can see, [this commit](https://github.com/digitalcoyote/chocolatey-packages/commit/890ec633080678d4cb0b58074b420212938ce7ba#diff-bc8d1e7356baad05e8420a6659103f47) did the fix and I can close this issue now. Right?

**Comment by simonsan** [comment]
Thank you as well from my side as well. :-)
